src/stan/analyze/mcmc/compute_potential_scale_reduction.hpp

```diff
@@ -31,6 +176,7 @@ namespace analyze {
  * @param sizes stores sizes of chains
  * @return potential scale reduction for the specified parameter
  */
+
 inline double compute_potential_scale_reduction(
```

Comment:
@bob-carpenter Should we be changing the old code at all? Do we want to always use the new rhat?